test/hooks.js

```diff
@@ -500,7 +500,7 @@ test('beforeRetry hook can cancel retries by returning `stop`', async t => {
 		}
 	});
 
-	await ky.get(server.url, {
+	await t.throwsAsync(ky.get(server.url, {
```

Comment:
Please add assertions about the thrown error. Otherwise, for all we know Ky is experiencing some kind of internal error.